multinet/__init__.py

```diff
@@ -37,6 +41,9 @@ def create_app(config: Optional[MutableMapping] = None) -> Flask:
     app.register_blueprint(downloaders.csv.bp, url_prefix="/api")
     app.register_blueprint(downloaders.d3_json.bp, url_prefix="/api")
 
+    app.register_blueprint(google.bp, url_prefix="/google")
```

Comment:
As an extension, we could even do `/user/oauth/google` since we may have more than 1 oauth provider – we could use github for example.

Reply:
Right, that was I meant in case it wasn't clear: `/user/oauth/` could be the namespace for these blueprints; `/user/oauth/google` would be the path for google logins. This would allow things like `/user/info` (what Girder calls `/user/me`) and other things like that to slot in cleanly.

Reply:
I like this, I can reorganize it.

Reply:
See da5729c, lmk what you guys think.

Reply:
Looks good. I think we will need swagger documentation for these endpoints right?

Reply:
Correct

Reply:
892b70e